base_tier_validation/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Base Tier Validation",
     "summary": "Implement a validation process based on tiers.",
-    "version": "12.0.3.4.0",
+    "version": "12.0.3.4.2",
```

Comment:
Do not change the version if no migrations are required, This simplifies the cherry-picks

Reply:
Ok, I will fix it. Thanks.